vm/src/gas/price_list.rs

```diff
@@ -0,0 +1,192 @@
+    #[inline]
+    pub fn on_ipld_get(&self, data_size: usize) -> i64 {
+        self.ipld_get_base + data_size as i64 * self.ipld_get_per_byte
+    }
+    /// Returns the gas required for storing an object
+    #[inline]
+    pub fn on_ipld_put(&self, data_size: usize) -> i64 {
+        self.ipld_put_base + data_size as i64 * self.ipld_put_per_byte
+    }
+    /// Returns the gas required for creating an actor
+    #[inline]
+    pub fn on_create_actor(&self) -> i64 {
+        self.create_actor_base + self.create_actor_extra
+    }
+    /// Returns the gas required for deleting an actor
+    #[inline]
+    pub fn on_delete_actor(&self) -> i64 {
+        self.delete_actor
+    }
+    /// Returns gas required for signature verification
+    #[inline]
+    pub fn on_verify_signature(&self, sig_type: SignatureType, plain_text_size: usize) -> i64 {
+        match sig_type {
+            SignatureType::BLS => (3 * plain_text_size + 2) as i64,
+            SignatureType::Secp256 => (3 * plain_text_size + 2) as i64,
+        }
+    }
+    /// Returns gas required for hashing data
+    #[inline]
+    pub fn on_hashing(&self, data_size: usize) -> i64 {
+        self.hashing_base + data_size as i64 * self.hashing_per_byte
+    }
+    /// Returns gas required for computing unsealed sector Cid
+    #[inline]
+    pub fn on_compute_unsealed_sector_cid(
+        &self,
+        _proof: RegisteredProof,
+        _pieces: &[PieceInfo],
+    ) -> i64 {
+        self.compute_unsealed_sector_cid_base
+    }
+    /// Returns gas required for seal verification
+    #[inline]
+    pub fn on_verify_seal(&self, _info: &SealVerifyInfo) -> i64 {
+        self.verify_seal_base
+    }
+    /// Returns gas required for PoSt verification
+    #[inline]
+    pub fn on_verify_post(&self, _info: &PoStVerifyInfo) -> i64 {
+        self.verify_post_base
+    }
+    /// Returns gas required for verifying consensus fault
+    #[inline]
+    pub fn on_verify_consensus_fault(&self) -> i64 {
+        self.verify_consensus_fault
+    }
+}
+
+impl Default for PriceList {
+    fn default() -> Self {
+        BASE_PRICES
+    }
+}
+
+const BASE_PRICES: PriceList = PriceList {
+    on_chain_message_base: 0,
+    on_chain_message_per_byte: 2,
+    on_chain_return_value_per_byte: 8,
+    send_base: 5,
+    send_transfer_funds: 5,
+    send_invoke_method: 10,
+    ipld_get_base: 10,
+    ipld_get_per_byte: 1,
+    ipld_put_base: 20,
+    ipld_put_per_byte: 2,
+    create_actor_base: 40,
+    create_actor_extra: 500,
+    delete_actor: -500,
+    hashing_base: 5,
+    hashing_per_byte: 2,
+    compute_unsealed_sector_cid_base: 100,
+    verify_seal_base: 2000,
+    verify_post_base: 700,
+    verify_consensus_fault: 10,
+};
+
+/// Returns gas price list by Epoch for gas consumption
+pub fn price_list_by_epoch(_epoch: ChainEpoch) -> PriceList {
+    // In future will match on epoch and select matching price lists when config options allowed
```

Comment:
Nit add TODO.

Reply:
There is nothing to do here, the protocol is not supporting different price tables yet (unless I am mistaken), this is just a feature